sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusSender.cs

```diff
@@ -257,7 +257,7 @@ protected ServiceBusSender()
             long sequenceNumber;
             try
             {
-                message.ScheduledEnqueueTimeUtc = scheduleEnqueueTime.UtcDateTime;
```

Comment:
I noticed that the parameter is scheduleEnqueueTime, whereas the property on the message is ScheduledEnqueueTime. This matches Track 1, but I wonder if we should update this to be consistent. I guess Scheduled is past tense, so maybe it was intentional to have them be different, but still seems odd. What do you think of doing ScheduledEnqueueTime for the parameter also?

Reply:
I agree with renaming the parameter as `ScheduledEnqueueTime` for consistency.